include/bluetooth/bluetooth.h

```diff
@@ -1153,6 +1153,9 @@ struct bt_le_per_adv_sync_term_info {
 
 	/** Advertiser SID */
 	uint8_t sid;
+
+	/** Cause of periodic advertising termination */
+	uint8_t err;
```

Comment:
Should we rename this `reason` instead of `err`, to be more consistent with the similar value for ACL connection failed/lost or ISO connection failed/lost or ISO sync failed/lost?

Reply:
Good hint, I'll rename it.